flow360/component/flow360_params/params_base.py

```diff
@@ -366,6 +372,39 @@ def _swap_key_in_nested_dict(cls, dictionary, key_to_replace, replacement_key):
 
         return dictionary
 
+    @classmethod
+    def _transform_optional_field(cls, model: dict, key: str):
```

Comment:
rename function to be more descriptive, eg:
`_generate_schema_for_optional_objects()`

Or maybe we can create a member schema class, where we can put such functions.